### ISSUE TITLE
Change README

### ISSUE THREAD

**Issue by anater** [juanparadox/react-tachyons#25]
We need to be more descriptive about this repo for newcomers!

**Comment by juanparadox** [comment]
Agreed! Shall we discuss this offline or flesh out the structure here?

**Comment by anater** [comment]
Hrmmmmmmmm. Let's discuss offline and then get it up here anyway. Unless you have some specific ideas you'd rather hash out in the issue.

**Comment by juanparadox** [comment]
Since we finished the components, we should probably start editing the readme. In case people want to contribute.

**Comment by anater** [comment]
Yeah, we could do that. What else is there to do before sharing it? This, rough documentation, and the website?
